autorag/chunker.py

```diff
@@ -46,12 +47,13 @@ def start_chunking(self, yaml_path: str):
 		input_modules, input_params = get_param_combinations(modules)
 
 		logger.info("Chunking Start...")
-		run_chunker(
-			modules=input_modules,
-			module_params=input_params,
-			parsed_result=self.parsed_raw,
-			trial_path=os.path.join(self.project_dir, trial_name),
-		)
+		for _ in tqdm(range(1), desc="Chunking Progress"):
+			run_chunker(
+				modules=input_modules,
+				module_params=input_params,
+				parsed_result=self.parsed_raw,
+				trial_path=os.path.join(self.project_dir, trial_name),
+			)
```

Comment:
I don’t figure out the reason why the tqdm range shown in here. It is not that different between `logger.info` about chunking done.